db/migrations/03_add_date_to_peeps.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE peeps ADD COLUMN date DATE NOT NULL;
```

Comment:
You should add this to db/migrations/01_create_chitter_table.sql (CREATE TABLE peeps(  id SERIAL PRIMARY KEY, message VARCHAR(60), name VARCHAR(20)), ;  date DATE NOT NULL)